_clients/data-prepper/get-started.md

```diff
@@ -9,15 +9,42 @@ nav_order: 1
 
 Data Prepper is an independent component, not an OpenSearch plugin, that converts data for use with OpenSearch. It's not bundled with the all-in-one OpenSearch installation packages.
 
-## 1. Install Data Prepper
+If you are migrating from Open Distro Data Prepper, visit the [Migrating from Open Distro](migrating_from_opendistro.md) page. Otherwise, please continue.
 
-To use the Docker image, pull it like any other image:
+## 1. Installing Data Prepper
 
-```bash
+There are two ways to install Data Prepper for running:
+
+1. Run the Docker image
+2. Build from source
```

Comment:
It is possible to install from `tar.gz` now. (Our repo is out-of-date on this).

I'm not sure we want to have "Build from source" in the opensearch.org documentation. It is mostly useful in certain situations (bleeding-edge, contributions).

Reply:
@dlvenable Do you have a link to the tar.gz file?

Reply:
The tar.gz changes with every build. I think the best solution is to point the customer to the download page: https://opensearch.org/downloads.html#data-prepper

That will have the latest version available.

Reply:
@Naarcha-AWS FYI, there is some new information here.

Reply:
@dlvenable Noted - I am going to make a separate issue to address the tar.gz installation instructions.